### PR TITLE
doc: mostly load-balancer to load balancer

### LINKED ISSUE #912: doc: Revise use of "load balancer" and "load-balancer"
Documentation lingo:
* Use "load balancer" as a noun--as in "...you request a load balancer,...".
* Use "load-balancer" as an adjective--as in, "...load-balancer service...".

Those two catch my eye when they aren't applied quite correctly.  I plan to apply that change throughout.

I'd like to reserve the right to expand the scope on lingo--and maybe style--as time allows.  I'm time-boxing this to a few hours.

I can't imagine any resistance, but I'm opening this issue to feel that out before I open a PR.  Please let me know if there's a reason *not* to make the change.

### PR DESCRIPTION
**Pull request by mikemckiernan**
Fixes #912.

### CODE REVIEW

File: website/content/_index.md [hunk 1]
```diff
@@ -16,23 +16,23 @@ explains what that implies.
 
 ## Why?
 
-Kubernetes does not offer an implementation of network load-balancers
+Kubernetes does not offer an implementation of network load balancers
 ([Services of type LoadBalancer](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/))
-for bare metal clusters. The implementations of Network LB that
+for bare-metal clusters. The implementations of network load balancer that
```
Comment (russellb) [inline website/content/_index.md:21]: I think this should be plural?

```suggestion
for bare-metal clusters. The implementations of network load balancers that
```

File: website/content/configuration/_index.md [hunk 2]
```diff
@@ -4,19 +4,19 @@ weight: 4
 ---
 
 MetalLB remains idle until configured. This is accomplished by
-creating and deploying a configmap into the same namespace
+creating and deploying a config map into the same namespace
```
Comment (russellb) [inline website/content/configuration/_index.md:7]: This is referencing a type of Kubernetes resource.  Maybe it should be `ConfigMap` ?
```suggestion
creating and deploying a `ConfigMap` into the same namespace
```

File: website/content/_index.md (hunk 1 above)
Comment (mikemckiernan) [reply website/content/_index.md:21]: That's def an oops.  Thank you for noticing and pointing it out.

File: website/content/configuration/_index.md (hunk 2 above)
Comment (mikemckiernan) [reply website/content/configuration/_index.md:7]: I favor lower case text without styling whenever possible.  Overwhelmingly, references to API objects like config maps, deployments, pods, and so on are more readable as plain text.  I acknowledge that this is often subjective.

Comment (russellb) [reply website/content/configuration/_index.md:7]: Thanks. I was mostly interested in your opinion and happy to go with what you suggest.

**Pull request merged**